<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CES Device Simulator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/overlay/Background.png</normaloff>res/overlay/Background.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="skin_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/overlay/Background.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_power">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/buttons/powerButton.png);
	border-radius: 20px;
}

QPushButton:pressed {
	image: url(:/buttons/power_on.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>510</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/buttons/delButton.png);
	border-radius: 20px;
}

QPushButton:pressed {
	image: url(:/buttons/del_press.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dur_3hrs">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/3hr Session.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dur_custom">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/User Session.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="batteryLevel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>34</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(82, 129, 22);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>79</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/buttons/saveButton.png);
	border-radius: 20px;
}

QPushButton:pressed {
	image: url(:/buttons/save_press.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_up">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/buttons/upButton.png);
	border-radius: 20px;
}

QPushButton:pressed {
	image: url(:/buttons/up_press.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/buttons/dnButton.png);
	border-radius: 20px;
}

QPushButton:pressed {
	image: url(:/buttons/dn_press.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ok">
    <property name="geometry">
     <rect>
      <x>438</x>
      <y>510</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/buttons/okButton.png);
	border-radius: 20px;
}

QPushButton:pressed {
	image: url(:/buttons/ok_press.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_home">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 20px;
	image: url(:/buttons/homeButton.png);
}

QPushButton:pressed {
	border-radius: 20px;
	image: url(:/buttons/home_press.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="connection_CES">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/CES Output.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="connection_audio">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Audio Input.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mode_L">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Left.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mode_R">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Right.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="powerLED">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>10</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="skin_menu">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/overlay/Menu.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="table_menu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ses_met">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/MET.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_subDelta">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Sub-Delta.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_delta">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Delta.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_theta">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Theta.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_alpha">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Alpha.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_beta1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Beta 1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_beta2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/Beta 2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ses_100Hz">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/100Hz.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="extra_tdcs">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>tDCS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="extra_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="extra_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="extra_200">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>2.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="extra_150">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>1.50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="extra_100">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>1.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="extra_075">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>0.75</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="extra_050">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>0.50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="extra_025">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(196, 196, 196);</string>
    </property>
    <property name="text">
     <string>0.25</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bar_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/8.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/7.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/6.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/5.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dur_20min">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	image: url(:/icons/20min Session.png);
	border-radius: 20px;
}

QPushButton:pressed {
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dur_45min">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/45min Session.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
